mobile/asa-go/src/App.tsx

```diff
@@ -160,6 +160,8 @@ const App = () => {
           setDate={setDateOfInterest}
           setTab={setTab}
           testId="asa-go-map"
+          runType={runType}
```

Comment:
Instead of passing `runType` and `runDatetime` as props, I would grab them from the `runParameterSlice` with the `selectRunParameter` selector.